modules/mobile/app/controllers/mobile/v0/community_cares_eligibility_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCaresEligibilityController < ApplicationController
```

Comment:
I would change this to `CommunityCare` singular because that's what we use elsewhere in the app.